### PR TITLE
use systemd modprobe service for speakup_soft

### PR DESCRIPTION
**Pull request by williamh**

### CODE REVIEW

File: services/systemd/espeakup.service.in [hunk 1]
```diff
@@ -2,13 +2,12 @@
 Description=Software speech output for Speakup
 Documentation=man:espeakup(8)
 Wants=systemd-udev-settle.service
```
Comment (alex19EP) [inline services/systemd/espeakup.service.in:4]:
```suggestion
Wants=modprobe@speakup_soft.service
```

File: services/systemd/espeakup.service.in [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
-After=systemd-udev-settle.service sound.target
+After=modprobe@speakup_soft.service systemd-udev-settle.service sound.target
```
Comment (alex19EP) [inline services/systemd/espeakup.service.in:5]:
```suggestion
After=modprobe@speakup_soft.service sound.target
```

**Review by alex19EP — CHANGES_REQUESTED** [review]
I think we can improve unit further. see my suggestions.

I need to test this. and also read the systemd-unit manual again.

**Comment by williamh** [comment]
@alex19EP  ping?

**Review by alex19EP — APPROVED** [review]

**Pull request merged**